app/services/frenet/shipping_info.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Frenet
+  class ShippingInfo
+    Error = Class.new(StandardError)
+    InvalidShippingInfoError = Class.new(Error)
+    PackageNotFound = Class.new(Error)
+
+    def initialize(shop)
+      @shop = shop
+    end
+
+    def find(package_code)
+      package = Vnda::Package.new(@shop).get(package_code)
+      service_name = package.dig('shipping_label') if package
+      return service_code(service_name) if service_name
+
+      raise PackageNotFound, "Package #{package_code} not found"
```

Comment:
Se retornar o pacote certamente o shipping_label estará preenchido

```suggestion      
      shipping_label = Vnda::Package.new(@shop).get(package_code)&.dig('shipping_label')
      raise PackageNotFound, "Package #{package_code} not found" unless shipping_label
      
      service_code(shipping_label)
```

Reply:
Ok, alteração feita